DataCapturing/Persistence/FileSupport.swift

```diff
@@ -0,0 +1,91 @@
+//
```

Comment:
This file contains the file handling in a protocol oriented programming style way.